<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>findForm</class>
 <widget class="QWidget" name="findForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>328</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Roboto</family>
    <italic>false</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>Поиск</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="btnHome">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../Загрузки/png-transparent-house-svg-vector-gray-house-gray-vector.png</normaloff>../../Загрузки/png-transparent-house-svg-vector-gray-house-gray-vector.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="rbtnMarka">
         <property name="text">
          <string>поиск по марке</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbtnSurn">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>поиск по владельцу</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbtnYear">
         <property name="text">
          <string> поиск по году</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Введите марку:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnSearch">
       <property name="text">
        <string>Найти</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../Загрузки/png-transparent-magnifying-glasses-illustration-magnifying-glass-icon-magnifier-cartoon-glasses-magnifier-thumbnail(1)-Photoroom.png-Photoroom.png</normaloff>../../Загрузки/png-transparent-magnifying-glasses-illustration-magnifying-glass-icon-magnifier-cartoon-glasses-magnifier-thumbnail(1)-Photoroom.png-Photoroom.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnHome</sender>
   <signal>clicked()</signal>
   <receiver>findForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
